test/spec/proxyBenchmark.js

```diff
@@ -8,11 +8,11 @@ if (typeof window === 'undefined') {
 }
 
 if (typeof jsonpatch === 'undefined') {
-  var jsonpatch = require('fast-json-patch');
```

Comment:
is there any reason why you removed this `var`?

Reply:
Yes, running tests in browser causes a collision because `jsonpatch` is defined and `var jsonpatch` is hoisted. And btw I added `var` and then removed it. So it wasn't there for a reason.

Reply:
ehh... I wish we have ES Modules in IE/Edge and FF w/o the flag. Could you add a small note in code comment about it?